application/slack/v0/api_functions.go

```diff
@@ -8,9 +8,9 @@ import (
 	"github.com/slack-go/slack"
 )
 
-func loopChannelListAPI(e *execution, isPublic bool, channelName string) (string, error) {
+func loopChannelListAPI(e *execution, channelName string) (string, error) {
 	var apiParams slack.GetConversationsParameters
-	setChannelType(&apiParams, isPublic)
+	apiParams.Types = append(apiParams.Types, "private_channel", "public_channel")
```

Comment:
If the `types` is always "private_channel" and "public_channel," we can just make it be constant when sending the request to the Slack API in `GetConversationsForUserContext()`.

Reply:
@donch1989 
Do you mean like these?
1. fork Slack package
2. modify the Slack package
3. Change the import source

Reply:
Except this, others are done. So, I request the review again!

Reply:
Oh.. sorry, I wrote the wrong function name.
I mean you can have a global variable and use it like this

```
var types []string{"private_channel", "public_channel"}

---

apiParams.Types = types
```